AndroidTest/gradle.properties

```diff
@@ -11,6 +11,12 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+# If SQLCipher-based datastore is required, use the sqlcipher parameter:
+# org.gradle.jvmargs=-Dtest.sqlcipher.passphrase=true
+# For running tests under only one package use 'test.run-packages'
+# TODO: Add comma option for running under multiple packages
+# Example for running test cases under 'datastore' package:
+# org.gradle.jvmargs=-Dtest.run-packages=datastore
```

Comment:
I was having trouble getting the prefix 'systemProp' to be recognized during debugging.  Will try again tomorrow.